### ISSUE TITLE
Classification CNN problem

### ISSUE THREAD

**Issue by benjisympa** [tiny-dnn/tiny-dnn#592]
Hi,
I follow the turorial (https://media.readthedocs.org/pdf/tiny-dnn/latest/tiny-dnn.pdf) and the doc (http://tiny-dnn.readthedocs.io/en/latest/how_tos/Train-network-with-your-dataset.html) to import images with open-cv and convert it in vec_t, it works but the network classify only on the first category.
For example with 2 images : 
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
start learning
end training.
accuracy:50% (1/2)
    *     0     1 
    0     1     1 
    1     0     0 
It's resized image in 32*32 and network is like that : 
net << conv<tan_h>(32, 32, 5, 1, 6)  // in:32x32x1, 5x5conv, 6fmaps
                << ave_pool<tan_h>(28, 28, 6, 2) // in:28x28x6, 2x2pooling
                << fc<tan_h>(14 * 14 * 6, 120)   // in:14x14x6, out:120
                << fc<identity>(120, 2);        // in:120,     out:10

assert(net.in_data_size() == taille * taille);

I dont' know why, have you got any idea ?

Thanks a lot.

**Comment by Randl** [comment]
>` fc(120, 2); // in:120, out:10`

I believe your output consists of two neurons (second number).

**Comment by benjisympa** [comment]
Thank you for your answer, but it's just a wrong comment.
I want to have 2 classes.
I found 2 others people with this unresolved problem : 
https://github.com/tiny-dnn/tiny-dnn/issues/28
http://stackoverflow.com/questions/40367232/error-with-tiny-dnn-for-pedestrian-recognition

**Comment by bhack** [comment]
Yes probably it is better to complete stackoverflow reply so that we don't lost knowledge
